Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -41,6 +42,10 @@ class PlaceCarouselViewController: UIViewController {
         didSet {
             // TODO: how do we make sure the user wasn't interacting?
             placeCarousel.refresh()
+
+            if oldValue.count == 0 {
```

Comment:
Yeah, I kinda figured that we would want to fine tune this when we start playing with the app and working out exactly what the logic should be